lib/rules/no-useless-escape.js

```diff
@@ -23,6 +23,12 @@ const VALID_STRING_ESCAPES = [
     "\r"
 ];
 
+const VALID_TEMPLATE_ESCAPES = VALID_STRING_ESCAPES.concat([
+    "$",
+    "{",
+    "}"
```

Comment:
Is `}` a valid template escape? I was thinking of something like `` `foo${bar\}` ``, but that's a SyntaxError.